bundles/org.openhab.binding.networkupstools/README.md

```diff
@@ -0,0 +1,127 @@
+# Network UPS Tools Binding
+
+The primary goal of the [Network UPS Tools](http://www.networkupstools.org/) (NUT) project is to provide support for power devices, such as uninterruptible power supplies (UPS), Power Distribution Units and Solar Controllers.
+
+Network UPS Tools (NUT) provides many control and monitoring features, with a uniform control and management interface.
+More than 100 different manufacturers, and several thousands of models are compatible.
+
+This binding lets you integrate NUT servers with openHAB.
+
+## Supported Things
+
+The binding can connect to multiple NUT instances.
+The thing is a `ups` thing.
+The thing supports a number of nut features out-of-the-box.
+And supports the option to configure additional channels to get other NUT variables.
+The thing queries the NUT server for the status of the UPS approximate every 3 seconds.
+And updates the status when a change happens.
```

Comment:
Consider combining with the previous sentence.

"The thing queries the NUT server for the status of the UPS approximate every 3 seconds and updates the status when a change happens.